tools/erc-repository-indexer/erc-contract-indexer/src/index.ts

```diff
@@ -0,0 +1,81 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import dotenv from 'dotenv';
+import { ByteCodeAnalyzer } from './services/byteCodeAnalyzer';
+import { ConfigService } from './services/config';
+import { ContractScannerService } from './services/contractScanner';
+import { RegistryGenerator } from './services/registryGenerator';
+
+dotenv.config();
+
+export const main = async () => {
```

Comment:
Eventually we'll want an acceptance test run on this i.e. use local node, push some contracts, run and confirm you got `x` ERC20s and `y` ERC721s

Reply:
Righttt I was wondering if I missed something for this PR. You're correct; a suite of acceptance tests is needed. I will add it to the EPIC and will get to it soon!

Reply:
Great.
Resolving to unblock PR